dashbuilder/dashbuilder-backend/dashbuilder-services/src/test/java/org/dashbuilder/migration/DashbuilderDataMigrationTest.java

```diff
@@ -244,7 +244,22 @@ public void cleanup() {
             if (uri.getScheme().equals("git")) {
                 JGitFileSystem fs = (JGitFileSystem) ioService.getFileSystem(uri);
                 fs.getGit().getRepository().getDirectory().delete();
+                deleteFileSystem(fs);
             }
         }
     }
+
+    private void deleteFileSystem(FileSystem fs) {
+        try {
+            FileUtils.deleteDirectory(
+                ((JGitFileSystem) fs).getGit()
+                .getRepository()
+                .getDirectory()
+                .getAbsoluteFile()
+                .getParentFile());
+
+        } catch (java.io.IOException e) {
+            e.printStackTrace();
```

Comment:
I suggest logging this exception instead of printing the stacktrace

Reply:
done